app/assets/scripts/views/explore.js

```diff
@@ -96,11 +97,16 @@ var Explore = React.createClass({
       return this.map.fitBounds(bounds);
     }
     if (nextProps.vprommsBbox !== this.props.vprommsBbox) {
-      bounds = nextProps.vprommsBbox[Object.keys(nextProps.vprommsBbox)[0]];
```

Comment:
The `vprommsBbox` continues to be an object with a the `vprommId` as a key:
![image](https://user-images.githubusercontent.com/1090606/31274683-ed89268e-aa8b-11e7-9c58-ac2ac9a7f4e2.png)

The bbox needs to be extracted from there, either here or in the reducer when getting the data.

Reply:
This is breaking the whole functionality. When fixing this the zoom works

Reply:
Result stored as a bbox directly on the reducer